diff --git a/node_modules/bigint-buffer/dist/node.js b/node_modules/bigint-buffer/dist/node.js
index 513168c..055cfdc 100644
--- a/node_modules/bigint-buffer/dist/node.js
+++ b/node_modules/bigint-buffer/dist/node.js
@@ -7,7 +7,6 @@ let converter;
         converter = require('bindings')('bigint_buffer');
     }
     catch (e) {
-        console.warn('bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)');
     }
 }
 /**
